### PR TITLE
Added the location of the export zone file

### PR DESCRIPTION
**Pull request by BWMerlin**
Added the location to where the export zone file can be found (C:\Windows\System32\DNS) and that the location is on the DNS server.  Also added a note that -FileName is a filename and not a filepath.  This should clarify how users can use this command and where they can locate the output.

### CODE REVIEW

File: docset/winserver2019-ps/dnsserver/Export-DnsServerZone.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ By default, the cmdlet places the file in the Domain Name System (DNS) directory
 PS C:\> Export-DnsServerZone -Name "western.contoso.com" -FileName "exportedcontoso.com"
 ```
 
-This command exports the zone named western.contoso.com to a file called exportedcontoso.com.
+This command exports the zone named western.contoso.com to a file called exportedcontoso.com located in C:\Windows\System32\DNS on the DNS server.  Note that -FileName is a filename not a filepath.
```
Comment (JohanFreelancer9) [inline docset/winserver2019-ps/dnsserver/Export-DnsServerZone.md:35]:
```suggestion
This command exports the zone named western.contoso.com to a file called exportedcontoso.com located in C:\Windows\System32\DNS on the DNS server. Note that -FileName is a filename, not a file path.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @BWMerlin, please commit my minor suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by dariomws — APPROVED** [review]
I think the same update must be applied to all Windows versions (from 2012 to 2022).

**Review by get-itips — CHANGES_REQUESTED** [review]
Tested this, ok, can you please also correct the description for `FileName` parameter? Thank you!

**Review by tseward — APPROVED** [review]

**Comment by BWMerlin** [comment]
> 
> 
> Tested this, ok, can you please also correct the description for `FileName` parameter? Thank you!

I *think* I have made the requested change to -FileName.  Please review.

**Review by get-itips — APPROVED** [review]

**Pull request merged**